deploy/intellabs/kafl/roles/fuzzer/templates/env.j2

```diff
@@ -1,8 +1,17 @@
-export WORKSPACE="{{ kafl_install_root }}"
-export KAFL_ROOT="{{ fuzzer_root }}"
+# source this file to activate the environment,
+# or use "make env" to start a sub-shell
+
+# components managed by ansible
+export KAFL_ROOT="{{ kafl_install_root }}"
 export QEMU_ROOT="{{ qemu_root }}"
 export LIBXDC_ROOT="{{ libxdc_root }}"
 export CAPSTONE_ROOT="{{ capstone_root }}"
 export RADAMSA_ROOT="{{ radamsa_root }}"
+
+# workspace defaults
+export WORKSPACE="{{ install_root }}"
```

Comment:
`install_root` is not defined in kafl, contrary to CCC.
in kafl, I renamed it to `kafl_install_root` to override it from CCC's playbook at import time.

You can use `playbook_dir/..` for the workspace root